scripts/build/ci-build/README.md

```diff
@@ -14,3 +14,5 @@ In order to build and publish the Grafana build Docker image, execute the follow
 docker build -t grafana/build-container:<VERSION> .
 docker push grafana/build-container:<VERSION>
 ```
+
+If running on an ARM chip (Apple M1/M2, etc.), be sure to add `--platform linux/amd64` to the `docker build` command and prepare for this to take a while (~4 hours for an initial build, faster thereafter due to caching)
```

Comment:
```suggestion
If you run Grafana on a machine that has ARM chip (Apple M1/M2, etc.), add `--platform linux/amd64` to the `docker build` command. It can take approximately four hours for an initial build to complete. Due to caching, subsequent builds take less time.
```

Reply:
hey @chri2547 - have made these changes but left out the bit about `If you run Grafana on...` since it's not about running grafana, it's about running this script in particular. hope that makes sense 👍

Reply:
@ashharrison90 , got it, in that case I would just say "If the script runs on a machine that has an ARM chip...".